samples/quote/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright © 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+ACONCLI ?= aconcli
+OPENSSL ?= openssl
+DOCKER  ?= docker
+GO      ?= go
+SERVER  ?= sampleserver
+
+all: server.json client/sampleclient
+
+server.json: server/Dockerfile signer.pem signer.cer | .acon/
+	$(DOCKER) build -t $(SERVER) -f $< $(PWD)/../..
+	$(ACONCLI) generate -i $(SERVER) $@
+	$(ACONCLI) sign -k signer.pem -c signer.cer $@
+
+%.pem:
+	$(OPENSSL) ecparam -genkey -name secp384r1 -out $@
+
+%.cer: %.pem
+	$(OPENSSL) req -x509 -sha384 -key $< -out $@ -outform der -subj /CN=self-signed-$<
+
+client/sampleclient:
+	$(GO) -C $(@D) build -v
```

Comment:
Not sure if it is better to build a static linked binary. For example, to build it in development environment and test in EMR environment - may cause link errors.